server_environment_data_encryption/models/server_env_mixin.py

```diff
@@ -56,7 +56,7 @@ def action_change_env_data_encrypted_fields(self):
             # We don't know which action we are using... take default one
             action = self.get_formview_action()
         else:
-            action = self.env["ir.actions.act_window"].browse(action_id).read()[0]
+            action = self.env["ir.actions.act_window"]._for_xml_id(action_id)
```

Comment:
@sebalix Not sure about this one.
`action_id` is the database id while `_for_xml_id` takes an `xml_id` as an input.

I guess using sudo in the read would be better

Reply:
Oups you are right, I was a bit fast...

Reply:
Fixed